README.md

```diff
@@ -169,8 +165,13 @@ const path2image = "..."
 
 const topResults = 5
 
-const result = await model.inference(path2image,topResults)
-
+const result = await model.inference(path2image, topResults)
+// or
+const result = await model.inference({
+  imgPath: path2image,
+  topK: topResults,
+  locale: 'zh', // will return label use specified locale, you can find all locales in https://github.com/luoye-fe/imagenet-labels-i18n
```

Comment:
great job thanks a lot!  
wouldn't be better if we use the list from this repository?
https://github.com/anishathalye/imagenet-simple-labels/blob/master/imagenet-simple-labels.json

then we can use a simple 

https://www.npmjs.com/package/language-translator
and get support to a lot of languages

Reply:
`imagenet-simple-labels.json`, for non-native English speakers, there is no difference, do you think that is more accurate? if so, we can use it

tried machine translation, will be some inaccuracy, must review manually, i think it can be added when someone needs it, and ask him to help check

actually, if really want to search using labels, high probability needs to use NLP